### PR TITLE
CLOUD-2790: Update verbiage of deactivation Modal

### PR DESCRIPTION
**Pull request by nmsrreabel**
image

Updating the verbiage of the deactivation modal based on discussions relating to the process

### CODE REVIEW

**Review by scthompsonopen — APPROVED** [review]

**Comment by Bonrob2** [comment]
Minor change: add a period at the end of the two steps.

File: plugin-core/src/main/ui/src/components/CloudPlugin.vue [hunk 1]
```diff
@@ -164,8 +164,12 @@ onMounted(async () => {
     :labels="labels"
   >
     <div style="width: 630px;">
-      Deactivating cloud services requires you to restart OpenNMS. After deactivation and restart, your data will persist to RRD storage.
-      You will no longer be able to access the data stored in the cloud or view it in the Portal.
+      Deactivating cloud services requires you to perform two extra tasks:
+      <ol type="1">
+        <li>Remove <em>/opt/opennms/etc/opennms.properties.d/hosted_tsdb.timeseries.properties</em> from your OpenNMS system</li>
```
Comment (Bonrob2) [inline plugin-core/src/main/ui/src/components/CloudPlugin.vue:169]:
```suggestion
        <li>Remove <em>/opt/opennms/etc/opennms.properties.d/hosted_tsdb.timeseries.properties</em> from your OpenNMS system</li>.
```

File: plugin-core/src/main/ui/src/components/CloudPlugin.vue [hunk 2]
```diff
@@ -164,8 +164,12 @@ onMounted(async () => {
...
+        <li>Restart OpenNMS</li>
```
Comment (Bonrob2) [inline plugin-core/src/main/ui/src/components/CloudPlugin.vue:170]:
```suggestion
        <li>Restart OpenNMS</li>.
```

**Review by Bonrob2 — COMMENTED** [review]
minor changes.

**Review by scthompsonopen — APPROVED** [review]

**Review by Bonrob2 — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**